federatorai/manifest.json

```diff
@@ -29,7 +29,6 @@
 	},
     "saved_views": {},
     "monitors": {
-		"Recommended Replica Number Reaches Max Limit": "assets/recommended_monitors/federatorai_recommended_replica_num.json",
```

Comment:
Is it expected to remove the monitor?

Reply:
yes, we decide to remove this monitor.

Reply:
Thanks for your answer, I'm merging it then!